Package.swift

```diff
@@ -18,22 +18,29 @@ let package = Package(
         .package(url: "https://github.com/attaswift/BigInt.git", from: "5.3.0"),
     ],
     targets: [
-        .target(name: "JWTKit", dependencies: [
-            .product(name: "Crypto", package: "swift-crypto"),
-            .product(name: "_CryptoExtras", package: "swift-crypto"),
-            .product(name: "X509", package: "swift-certificates"),
-            .product(name: "BigInt", package: "BigInt"),
-        ],
-        swiftSettings: [
-            .enableUpcomingFeature("ConciseMagicFile"),
-        ]),
+        .target(
+            name: "JWTKit",
+            dependencies: [
+                .product(name: "Crypto", package: "swift-crypto"),
+                .product(name: "_CryptoExtras", package: "swift-crypto"),
+                .product(name: "X509", package: "swift-certificates"),
+                .product(name: "BigInt", package: "BigInt"),
+
+            ],
+            swiftSettings: [
+                .enableExperimentalFeature("StrictConcurrency"),
+            ]
+        ),
         .testTarget(
             name: "JWTKitTests",
             dependencies: [
                 "JWTKit",
             ],
             resources: [
                 .copy("TestVectors"),
+            ],
+            swiftSettings: [
+                .enableUpcomingFeature("ConciseMagicFile"),
```

Comment:
We should add strict Sendable checking to the tests as well just to prepare for Swift 6